api/src/main/java/org/motechproject/nms/api/web/OpsController.java

```diff
@@ -80,4 +92,59 @@ public void startUpkeep() {
         LOGGER.info("/upkeep");
         eventRelay.sendEventMessage(new MotechEvent(KilkariConstants.SUBSCRIPTION_UPKEEP_SUBJECT));
     }
+
+    @RequestMapping(value = "/addFlw",
```

Comment:
nitpick, but since the method creates or updates, shouldn't the URL endpoint reflect that?